COMET.Web.Common/Components/ParameterTypeEditors/QuantityKindParameterTypeEditor.razor

```diff
@@ -30,9 +30,9 @@
                BindValueMode="@this.BindValueMode">
     </DxTextBox>
     
-    @if (this.ViewModel.Parameter.Scale != null)
+    @if (this.ViewModel.ParameterType.DefaultScale != null)
     {
-        <span class="content-scale">[@this.ViewModel.Parameter.Scale.ShortName]</span>
+        <span class="content-scale">[@this.ViewModel.ParameterType.DefaultScale.ShortName]</span>
```

Comment:
The current scale may not be the Default one. Needs to be the Parameter.Scale property. When it's inside a compoundParameterType, make sure to correctly pass the ParameterTypeComponent